frontend/src/pages/profile/UserProfileById.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+    import { derived } from "svelte/store";
+    import type { UserDTO } from "../../../../mock-backend/mocks/models/user";
+    import Page from "../../layout/Page.svelte";
+    import { _ } from "../../services/i18n";
+    import {
+        generalUsersStore,
+        loadAndSetUser,
+    } from "../../stores/generalUsersStore";
+    import UserProfile from "./components/UserProfile.svelte";
+
+    type UserId = UserDTO["id"];
+    export let userId: UserId;
+
+    loadAndSetUser(userId);
+    const userStore = derived(generalUsersStore, (store) => store[userId]);
```

Comment:
a derived store is a bit overkill here, a reactive statement would be sufficient:

```
$: theUser = $generalAllUsersStore[userId]
```

because of the `$:` statement the expression gets reevaluated whenever `generalAllUsersStore` and/or `userId` changes.

Reply:
I have changed accessing the user store here completely so that the function loading it returns a store showing the data and a store for error handling

Reply:
What do you think? @seriousManual